.github/workflows/quarto-publish.yml

```diff
@@ -0,0 +1,37 @@
+name: quarto-publish
+on:
+  push:
+    branches: dev
+jobs:
+  build-deploy:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@dev # checkout 'dev' branch
```

Comment:
hi @zander-prinsloo, This is causing errors. 

I think you can safely remove it. Since we are not going to use github actions, you can remove this file so that the PR does not trigger any error message. 

Thanks